cmd/rookd/api.go

```diff
@@ -31,26 +31,27 @@ import (
 )
 
 var apiCmd = &cobra.Command{
-	Use:    "api",
-	Short:  "Runs the Rook REST API service",
-	Hidden: true,
+	Use:   "api",
+	Short: "Runs the Rook API service",
 }
 var apiPort int
+var repoPrefix string
 
 func init() {
-	apiCmd.Flags().IntVar(&apiPort, "api-port", 0, "port on which the api is listening")
+	apiCmd.Flags().IntVar(&apiPort, "port", 0, "port on which the api is listening")
+	apiCmd.Flags().StringVar(&repoPrefix, "repo-prefix", "quay.io/rook", "the repo from which to pull images")
```

Comment:
this might fix the issue i saw earlier this week while testing for v0.3.1, where pods created by the API server didn't get the private repo prefix and failed to launch?